essentials-framework/Essentials DM/Essentials_DM/AirMedia/AirMediaController.cs

```diff
@@ -263,6 +263,29 @@ public int NumberOfComPorts
         #endregion
 
 
+
+        #region IRoutingNumeric Members
+
+        public void ExecuteNumericSwitch(ushort input, ushort output, eRoutingSignalType signalType)
+        {
+            if ((signalType & eRoutingSignalType.Video) != eRoutingSignalType.Video) return;
+            if (Enum.IsDefined(typeof (AmX00DisplayControl.eAirMediaX00VideoSource), input))

```

Comment:
One more change...can you invert this if and add a debug statement that prints the value at debug level 2. That way, if a switch isn't made, there's a traceable reason why.

Reply:
Done